app/views/runs/_title.slim

```diff
@@ -45,6 +45,14 @@ h5
             span Claim
         - else
           button.btn.btn-outline-secondary disabled=true Sign In to Claim
+    - if can?(:create, RunLike)
```

Comment:
If I understand this correctly, this means the like counter will not be shown at all if logged out?  If that's the case I would argue that this should still be shown, just not have any functionality, or clicking it should prompt the user to login.